webapp/src/cljc/lipas/utils.cljc

```diff
@@ -9,6 +9,12 @@
    #?(:cljs [goog.string :as gstring])
    #?(:cljs [goog.string.format])))
 
+
+(defn ->sortable-name [s]
```

Comment:
This will blow up with `nil`. It might be OK for this particular use-case where we know that sports-site name is a mandatory field (enforced by specs). For generic usage we can decide whether: 
- we treat nil as an error and let it throw
- ..or handle `nil` gracefully and return an empty string or something else that can be sorted

What do you think makes most sense here?

Reply:
Added handling for nil and a test case. Return an empty string like they do in `clojure.core/str`